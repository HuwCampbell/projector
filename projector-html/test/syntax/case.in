case foo of
  Abc ->
    "def"
  Ghi ->
    "jkl"
  Mno ->
    "pqr"
  _ ->
    "stu"
