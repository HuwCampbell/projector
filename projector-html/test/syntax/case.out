Template
  ()
  Nothing
  (TECase
     ()
     (TEVar () TId { unTId = "foo" })
     (TAlt
        ()
        (TPCon () TConstructor { unTConstructor = "Abc" } [])
        (TEString () (TIString () [ TStringChunk () "def" ])) :|
        [ TAlt
            ()
            (TPCon () TConstructor { unTConstructor = "Ghi" } [])
            (TEString () (TIString () [ TStringChunk () "jkl" ]))
        , TAlt
            ()
            (TPCon () TConstructor { unTConstructor = "Mno" } [])
            (TEString () (TIString () [ TStringChunk () "pqr" ]))
        ]))